src/elements/ts-main/ts-dashboard.html

```diff
@@ -940,14 +940,14 @@
                   var getreg = App.keyManager.getRegistrationInfo.bind(App.keyManager, deviceId),
                       genreg = App.keyManager.generateRegistrationInfo.bind(App.keyManager, deviceId),
                       rereg = function () {
-                          return App.userManager.unregister(user, deviceId).then(genreg);
+                          return App.userManager.unregister(user).then(genreg);
```

Comment:
The deviceId is used in the other two methods and we may need to dig into it, so we can avoid introducing hidden bugs somewhere we haven't tested